go.mod

```diff
@@ -18,9 +18,9 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/ncw/swift v1.0.47
-	github.com/nspcc-dev/neo-go v0.98.2
-	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.4.0.20220628114722-ab4d1e34a8ac
-	github.com/nspcc-dev/tzhash v1.5.2
+	github.com/nspcc-dev/neo-go v0.99.1
```

Comment:
Shouldn't we update to 99.2?

Reply:
SDK doesn't support neo-go v0.99.2 yet https://github.com/nspcc-dev/neofs-sdk-go/pull/328

Reply:
We don't use rpc client from `neo-go` which was changed in v0.99.2 so it doesn't matter.